portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,21 +26,58 @@
 import com.google.appengine.api.datastore.PreparedQuery;
 import com.google.appengine.api.datastore.Query;
 import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.appengine.api.datastore.FetchOptions;
+import java.util.ArrayList; 
+import java.util.List;
+import com.google.gson.Gson;
+import com.google.sps.data.Task;
+
+
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-/**
+  private int quantity = 5; 
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("data/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+
+    // Create a new query
+    Query query = new Query("Task");    
+
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    PreparedQuery results = datastore.prepare(query);
+
+    // Get query string
+    //int limit = Integer.parseInt(request.getQueryString()); 
+
+    // Set query limit to control maximum number of comments
+    List<Entity> sentence = results.asList(FetchOptions.Builder.withLimit(quantity));
+    List<Task> tasks = new ArrayList<>();
+    for (Entity entity : sentence) {
+        long id = entity.getKey().getId();
+        String animal = (String) entity.getProperty("animal");
+        String verb = (String) entity.getProperty("verb");
+        String adj = (String) entity.getProperty("adj");
+
+        Task task = new Task(id, animal, verb, adj);
+        tasks.add(task);
+
+    }
+
+    response.setContentType("application/json;");
+    String json = new Gson().toJson(tasks);
+    response.getWriter().println(json);
   }
-**/ 
 
    @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    
+    quantity = Integer.parseInt(request.getParameter("quantity"));
+    System.out.println(quantity);
```

Comment:
I assume you use this println to debug? Please either change the message printed to something more meaningful or remove it.